### PR TITLE
Seabeta 375 update cookie banner library

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: GovUkDesignSystem/GovUkDesignSystemComponents/CookieBannerViewModel.cs [hunk 1]
```diff
@@ -1,25 +1,131 @@
...
+    
+    /// <summary>
+    /// 	<b>Required</b>. The different messages you can pass into the cookie banner.
+    ///     For example, the cookie message and the confirmation message.
+    /// </summary>
+    public List<CookieBannerMessageViewModel> Messages { get; set; }
 }
 
-public enum BannerState
+public class CookieBannerMessageViewModel : IHtmlText
```
Comment (DanCorderSoftwire) [inline GovUkDesignSystem/GovUkDesignSystemComponents/CookieBannerViewModel.cs]: Please put `CookieBannerMessageViewModel` and `CookieBannerMessageActionViewModel` in separate files in the SubComponents folder to match the cshtml

**Comment by ghost** [comment]
Reference implementation:
https://github.com/alphagov/govuk-frontend/blob/main/src/govuk/components/cookie-banner/template.njk

File: GovUkDesignSystem/GovUkDesignSystemComponents/CookieBanner.cshtml [hunk 2]
```diff
@@ -1,78 +1,12 @@
...
-                        Value = "hide"
-                    })
-                }
-            </div>
-        </div>
-
-    </form>
-}
\ No newline at end of file
+<div class="govuk-cookie-banner @Model.Classes" data-nosnippet role="region" aria-label="@(Model.AriaLabel ?? "CookieBanner")"
```
Comment (DanCorderSoftwire) [inline GovUkDesignSystem/GovUkDesignSystemComponents/CookieBanner.cshtml]: `CookieBanner` -> `Cookie banner`

**Review by DanCorderSoftwire — CHANGES_REQUESTED** [review]

**Review by DanCorderSoftwire — COMMENTED** [review]
Looks good

**Pull request merged**